submissions/jeffersonMormah/domManipulation/contact.js

```diff
@@ -0,0 +1,45 @@
+let formContact = document.getElementById("contact-page");
+let fullName = document.getElementById("full-name");
+let email = document.getElementById("email");
+let contactSubject = document.getElementById("subject");
+let contactMessage = document.getElementById("message");
+let validName = /^[A-Za-z]/;
+let validEmail = /^+([]?+)*@+([-]?+)*({2,6})+$/;
+let validSubject = /^[A-Za-z]/;
+let validMessage = /^[A-Za-z]/;
```

Comment:
`validSubject`, `validMessage` and `validName` have the same value. It better to just use one of them